src/app/components/faq/faq.component.html

```diff
@@ -0,0 +1,34 @@
+<app-header-static></app-header-static>
```

Comment:
PR name look completely different with this changes :o 

We need to wait till surge deployed URL, to check the working copy.